examples/bri-3/src/bri/communication/agents/mockBpiMessagesStorage.agent.ts

```diff
@@ -1,44 +1,34 @@
+import { Mapper } from '@automapper/core';
+import { InjectMapper } from '@automapper/nestjs';
 import { Injectable, NotFoundException } from '@nestjs/common';
-import Mapper from '../../utils/mapper';
 import { NOT_FOUND_ERR_MESSAGE } from '../api/err.messages';
 import { BpiMessage } from '../models/bpiMessage';
-import { getType } from 'tst-reflect';
 
 @Injectable()
 export class MockBpiMessageStorageAgent {
-  constructor(private readonly mapper: Mapper) {}
-
   private bpiMessagesStore: BpiMessage[] = [];
 
   async getBpiMessageById(id: string): Promise<BpiMessage> {
     const bpiMessage = this.bpiMessagesStore.find((bp) => bp.id === id);
     if (!bpiMessage) {
       throw new NotFoundException(NOT_FOUND_ERR_MESSAGE);
     }
-    return this.mapper.map(bpiMessage, getType<BpiMessage>()) as BpiMessage;
+    return bpiMessage;
   }
 
   async getAllBpiMessages(): Promise<BpiMessage[]> {
-    return Promise.resolve(
-      this.bpiMessagesStore.map(
-        (bpiMessage) =>
-          this.mapper.map(bpiMessage, getType<BpiMessage>()) as BpiMessage,
-      ),
-    );
+    return Promise.resolve(this.bpiMessagesStore);
   }
 
   async createNewBpiMessage(bpiMessage: BpiMessage): Promise<BpiMessage> {
-    const createdBp = this.mapper.map(
-      new BpiMessage(
-        bpiMessage.id,
-        bpiMessage.from,
-        bpiMessage.to,
-        bpiMessage.content,
-        bpiMessage.signature,
-        bpiMessage.type,
-      ),
-      getType<BpiMessage>(),
-    ) as BpiMessage;
+    const createdBp = new BpiMessage(
```

Comment:
`createdBp`? does not make a lot of sense should it not be `createBpiMessage`?

Reply:
done